src/Stryker.CLI/Stryker.CLI.UnitTest/StrykerCLITests.cs

```diff
@@ -306,6 +306,38 @@ public void StrykerCLI_OnMutationScoreBelowThresholdBreak_ShouldReturnExitCode1(
             Assert.Equal(1, result);
         }
 
+        [Fact]
+        public void StrykerCLI_OnMutationScoreEqualToNullAndThresholdBreakEqualTo0_ShouldReturnExitCode0()
+        {
+            var mock = new Mock<IStrykerRunner>(MockBehavior.Strict);
+            StrykerOptions options = new StrykerOptions(thresholdBreak: 0);
+            StrykerRunResult strykerRunResult = new StrykerRunResult(options, null);
+            mock.Setup(x => x.RunMutationTest(It.IsAny<StrykerOptions>())).Returns(strykerRunResult).Verifiable();
+
+            var target = new StrykerCLI(mock.Object);
+            int result = target.Run(new string[] { });
+
+            mock.Verify();
+            Assert.Equal(0, target.ExitCode);
```

Comment:
It would be nice to use shouldly here too. But I now see that we neither did above here. Feel free to edit or keep it the same :)